test/test.py

```diff
@@ -658,6 +658,14 @@ def testGet(self):
         self.assertEqual([['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc2'))
         self.assertEqual([['f1', 'some valid content dd1', 'f2', 'this is sample text ff1'], ['f1', 'some valid content dd2', 'f2', 'this is sample text ff2']], client.get('doc1', 'doc2'))
 
+    def testConfig(self):
+        client = self.getCleanClient('idx')
```

Comment:
Thanks again for your contributions.
Can you please add a test for TIMEOUT before setting it and a test for a text-based config option such as timeoutpolicy or gcpolicy.